packages/ethtool/ethtool.spec

```diff
@@ -0,0 +1,33 @@
+Name:           %{_cross_os}ethtool
+Epoch:          2
+Version:        6.2
+Release:        1%{?dist}
+Summary:        Settings tool for Ethernet NICs
+License:        GPL-2.0-only AND GPL-2.0-or-later
+URL:            https://www.kernel.org/pub/software/network/ethtool/
+Source0:        https://www.kernel.org/pub/software/network/ethtool/ethtool-%{version}.tar.xz
+BuildRequires:  %{_cross_os}glibc-devel
+BuildRequires:  %{_cross_os}libmnl-devel
+Conflicts:      filesystem < 3
+
+%description
+This utility allows querying and changing settings such as speed,
+port, auto-negotiation, PCI locations and checksum offload on many
+network devices, especially of Ethernet devices.
+
+%prep
+%setup -n ethtool-%{version}
+
+%build
+%cross_configure
+%make_build
+%install
+%make_install
+
+%files
+%license COPYING LICENSE
+%{_cross_attribution_file}
+%{_cross_sbindir}/ethtool
+%exclude %{_cross_libdir}
```

Comment:
huh, does this package provide a shared library?

Reply:
No, that would be an over-greedy copy-paste from the other specfile I was grabbing. I'll remove.